{{ cookiecutter.project_name }}/docker-compose.yml

```diff
@@ -72,12 +66,50 @@ services:
     volumes:
       - ./webapp:/app
 
+  webapp:
+    build: "./webapp"
+    image: "{{ cookiecutter.project_name }}_webapp:dev"
+    restart: "on-failure"
+    ports:
+     - "8000:8000"
+    depends_on:
+      db:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      migrate:
+        condition: service_completed_successfully
+      collectstatic:
+        condition: service_completed_successfully
+    environment:
+      DJANGO_SETTINGS_MODULE: "${DJANGO_SETTINGS_MODULE}"
+      DJANGO_SECRET_KEY: "${DJANGO_SECRET_KEY}"
+      POSTGRES_HOST: "${POSTGRES_HOST}"
+      POSTGRES_DB: "${POSTGRES_DB}"
+      POSTGRES_USER: "${POSTGRES_USER}"
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      REDIS_URL: "${REDIS_URL}"
+    volumes:
+     - ./webapp:/app
+    healthcheck:
+      test: "curl --fail http://localhost:8000/?healthcheck || exit 1"
+      interval: 15s
+      timeout: 15s
+      retries: 4
```

Comment:
I guess since the initial app is small, it starts up quickly. Later, when the app grows and takes longer to start, you could add the `start_period` parameter.